### PR TITLE
Add example sizes, upgrade VM, and add labels for slack

### PR DESCRIPTION
**Pull request by arramos84**

### CODE REVIEW

File: charts/asserts/large.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    resources:
+      limits:
+        memory: 1Gi
+      requests:
+        cpu: 1
+        memory: 1Gi
+
+tsdb:
+  server:
+    retentionPeriod: 90d
```
Comment (xujiaxj) [inline charts/asserts/large.yaml:20]: What is this `90d` retention period?

Comment (arramos84) [reply charts/asserts/large.yaml:20]: The tsdb retention time. Should we make it longer? Also, I can make it match the graph.

Comment (arramos84) [reply charts/asserts/large.yaml:21]: I've added the retention to match the graph. We will have to do some re-work to have a single global value, but this is ok for now.

Comment (xujiaxj) [reply charts/asserts/large.yaml:21]: 90 days sounds fine. I was curious why it differs from the 30 days you mentioned earlier.

**Review by xujiaxj — APPROVED** [review]

File: charts/asserts/large.yaml (hunk 1 above)
Comment (arramos84) [reply charts/asserts/large.yaml:21]: I made the examples 90days bc it's more realistic for an actual deployment. Default values are mostly to just get up and running.

**Pull request merged**